src/app/src/mainwindow.cpp

```diff
@@ -248,7 +250,7 @@ void MainWindow::on_frameRateEdit_valueChanged(double value) {
         m_fps = value;
         m_expSettings.expTimeMS = (1 / m_fps) * 1000;
         m_expSettings.frameCount = m_duration * m_fps;
-        m_camera->SetupExp(m_expSettings);
+        m_camera->UpdateExp(m_expSettings);
```

Comment:
we might not even need to call this here now, since we aren't updating the camera settings while live view is running it only really needs to get called when the acquisition is started